.cloud/azuredeployBatchEndpoint.json

```diff
@@ -0,0 +1,228 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "randomId": {
+            "defaultValue": "j",
+            "type": "String",
+            "metadata": {
+                "description": "Specifies the name of the Azure Machine Learning workspace."
+            }
+        },
+        "workspaceName": {
```

Comment:
can we follow command job for ws and other entity name rather than randomID
so who ever using this repo wont have multiple ws and dependencies created just for running different samples ?